### PR TITLE
Intégration : ajout d'un paramètre `id` à l'index des rdvs

### PR DESCRIPTION
**Pull request by victormours**
# Contexte

Dans le contexte de l'intégration avec Démarches Simplifiées, ils vont enregistrer la liste des ids des rendez-vous liés à une démarche.
Il leur faut ensuite un endpoint pour récupérer la liste des rendez-vous liés à cette démarche au moment où ils chargent la page de détails de la démarche.

En venant chercher les infos de manière synchrone, ça évite les problèmes de synchronisation (les nerds qui aiment le cap theorem diraient qu'on échange de l'availability contre de la consistency), et ça leur permet d'afficher les infos à jour de tous les rendez-vous.

Il leur faut donc pouvoir récupérer une liste de rendez-vous.

# Solution

Petit choix discutable : je fais déborder la permissivité d'activerecord en acceptant soit un entier soit un tableau d'entier pour ce nouveau paramètre. C'est un peu excentrique, mais tellement pratique.

### CODE REVIEW

File: swagger/v1/api.json [hunk 1]
```diff
@@ -2824,6 +2824,13 @@
               "type": "integer"
             }
           },
+          {
+            "name": "id",
+            "in": "query",
+            "description": "Filtre pour obtenir uniquement les rendez-vous dont l'id est dans cette liste. Vous pouvez passer soit un tableau d'entier, soit un seul entier pour obtenir un seul rdv",
```
Comment (francois-ferrandis) [inline swagger/v1/api.json]: Il n'existe pas de façon standard de passer un tableau de valeurs via une querystring en HTTP GET ([cette réponse SO](https://stackoverflow.com/a/9547490) explique bien la situation), donc je pense qu'on peut préciser que le format attendu est `id[]=1234&id[]=5678`. Idéalement on peut ajouter un exemple dans les tests mais je sais que c'est relou et ça peut créer du bruit pour juste documenter un edge case, donc je te laisse juge.

On pourrait aussi s'assurer de supporter tous les formats, mais non, il vaut mieux qu'on dise qu'on supporte juste le format Rails standard.

**Review by francois-ferrandis — APPROVED** [review]
Petite suggestion facultative, sinon c'est nickel ! :rocket:

File: swagger/v1/api.json (hunk 1 above)
Comment (victormours) [reply swagger/v1/api.json]: tu as complètement raison, j'ajoute ça
